src/AWS.Deploy.CLI/ConsoleUtilities.cs

```diff
@@ -410,9 +410,9 @@ public void DisplayValues(Dictionary<string, object> objectValues, string indent
                         _interactiveService.WriteLine($"{indent}{key}: {stringValue}");
                     }
                 }
-                else if (value is bool boolValue)
+                else if(value != null)
                 {
-                    _interactiveService.WriteLine($"{indent}{key}: {boolValue}");
+                    _interactiveService.WriteLine($"{indent}{key}: {value}");
                 }

```

Comment:
do we want to open this up to accept other data types if they are not supported?

Reply:
I don't understand why. The code is the same in each block and it triggers bugs when new types are added and we didn't know to update all of the places it matters. For example this code was already broken not showing ints because who ever added ints didn't know to update this block of code.